legacy_publisher/dita_publisher.py

```diff
@@ -64,5 +66,7 @@ def publish_regions(regions=None):
     if not isExist:
         os.makedirs(dita_dir)
 
+    copy_tree(sourcepath+"/PlatformData/content", dita_dir+"/content")
```

Comment:
let´s not copy over the content directory.  It may not be called content., and we don´t necessarily need all of the files.  Let´s get the URL of the image, and copy the image to the destination.